pkg/cmd/server/api/types.go

```diff
@@ -14,12 +14,15 @@ const (
 )
 
 var (
-	KnownKubernetesAPILevels   = []string{"v1beta1", "v1beta2", "v1beta3", "v1"}
-	KnownOpenShiftAPILevels    = []string{"v1beta1", "v1beta3", "v1"}
-	DefaultKubernetesAPILevels = []string{"v1beta3", "v1"}
-	DefaultOpenShiftAPILevels  = []string{"v1beta3", "v1"}
-	DeadKubernetesAPILevels    = []string{"v1beta1", "v1beta2"}
-	DeadOpenShiftAPILevels     = []string{"v1beta1"}
+	KnownKubernetesAPILevels             = []string{"v1beta1", "v1beta2", "v1beta3", "v1"}
+	KnownOpenShiftAPILevels              = []string{"v1beta1", "v1beta3", "v1"}
+	KnownOpenShiftStorageVersionLevels   = []string{"v1beta1", "v1beta3", "v1"}
```

Comment:
Known levels should be https://github.com/openshift/origin/blob/master/pkg/api/latest/latest.go#L31